Dockerfile-windows-nano

```diff
@@ -0,0 +1,10 @@
+FROM microsoft/dotnet:2.1.403-sdk-nanoserver-1803
+
+ARG CAKE_VERSION=0.30.0
+
+# Windows doesn't currently support setting PATH with ENV
+RUN setx PATH "C:\Program Files\dotnet;c:\cake;%PATH%"
+
+RUN dotnet --info \
+	& dotnet tool install Cake.Tool --version %CAKE_VERSION% --tool-path c:\cake
```

Comment:
Shouldn't this be `&&` to ensure only execute if previous successful?

Reply:
You learn something new everyday - I'll fix that :)